src/TMG.Aimsun/AimsunTool.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using XTMF;
+
+namespace TMG.Aimsun
+{
+    [ModuleInformation(Description = @"The aimsun tool to run it takes an aimsun and a list of all the arguments you 
+    need to run it ")]
```

Comment:
With the @ string, all of spaces on line 7 will also be copied to the documentation.  You might want to align that on the left or just concatenate the string by parts.  The compiler will automatically add it all together.